core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -264,6 +275,24 @@ trait WorkspaceComponent {
       } yield workspace
     }
 
+    def listWorkspaceGoogleProjectsWithIncorrectBillingAccounts(): ReadAction[Seq[(GoogleProjectId, Option[RawlsBillingAccountName])]] = {
+      val query = for {
+        billingProject <- rawlsBillingProjectQuery if !billingProject.invalidBillingAccount
```

Comment:
why do we need `if !billingProject.invalidBillingAccount` ?

Reply:
We exclude records with `billingProject.invalidBillingAccount` because we know we will fail to set those billing accounts on projects.

There is not a bug on 283 because that is actually slick code that is translated to sql. All the conditionals in this for comprehension all boil down to join or where clauses.